From 02be7bf52341d0bd73ff1c3fde9f47167a37fa76 Mon Sep 17 00:00:00 2001
From: Michael Campion <michael.campion@emutex.com>
Date: Thu, 11 Jul 2019 14:58:04 +0000
Subject: [PATCH 23/30] upboard pinctrl: drop regmap patches for UP Xtreme

Since BIOS release UPW1AM0N (07/05/2019) for UP Xtreme, the FPGA pinctrl
configuration registers are intialised by the BIOS and regmap patches to set
default values can now be removed.
---
 drivers/mfd/upboard-fpga.c        | 1 -
 drivers/pinctrl/pinctrl-upboard.c | 8 --------
 2 files changed, 9 deletions(-)

diff --git a/drivers/mfd/upboard-fpga.c b/drivers/mfd/upboard-fpga.c
index 42e6f242e92a..532b58f0aed4 100644
--- a/drivers/mfd/upboard-fpga.c
+++ b/drivers/mfd/upboard-fpga.c
@@ -348,7 +348,6 @@ static const struct dmi_system_id upboard_dmi_table[] __initconst = {
 			DMI_EXACT_MATCH(DMI_BOARD_NAME, "UP-WHL01"),
 			DMI_EXACT_MATCH(DMI_BOARD_VERSION, "V0.1"),
 		},
-		.driver_data = (void *)UPFPGA_QUIRK_UNINITIALISED,
 	},
 	{ },
 };
diff --git a/drivers/pinctrl/pinctrl-upboard.c b/drivers/pinctrl/pinctrl-upboard.c
index f9bd3d157503..2ee1085e60db 100644
--- a/drivers/pinctrl/pinctrl-upboard.c
+++ b/drivers/pinctrl/pinctrl-upboard.c
@@ -839,14 +839,6 @@ static const struct dmi_system_id upboard_dmi_table[] __initconst = {
 		},
 		.driver_data = (void *)&upboard_up2_bios_info_v0_3,
 	},
-	{
-		.matches = { /* UP Xtreme */
-			DMI_EXACT_MATCH(DMI_SYS_VENDOR, "AAEON"),
-			DMI_EXACT_MATCH(DMI_BOARD_NAME, "UP-WHL01"),
-			DMI_EXACT_MATCH(DMI_BOARD_VERSION, "V0.1"),
-		},
-		.driver_data = (void *)&upboard_up_bios_info_dvt,
-	},
 	{ },
 };
 
-- 
2.17.1

